src/lib/prisma.ts

```
@@ -0,0 +1,13 @@
+import { PrismaClient } from "@/generated/prisma";
+
+declare global {
+  var prisma: PrismaClient | undefined;
+}
+
+export const prisma =
```

Biome (2.1.2)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)